layouts/partials/header/header.html

```diff
@@ -18,7 +18,7 @@
   {{ end -}}
 
     <!-- Site title -->
-    <a class="navbar-brand me-auto me-lg-3" href="{{ "/" | relLangURL }}">{{ .Site.Title }}</a>
+    <a class="navbar-brand me-auto me-lg-3" href="{{ .Site.BaseURL | absURL }}">{{ .Site.Title }}</a>
```

Comment:
Please use `href="{{ site.BaseURL | relLangURL }}"` instead. Your current code breaks multilingual sites.

Reply:
@lperdereau Do you want to update this PR as suggested?